assets/rpip-10/expense_3.md

```diff
@@ -0,0 +1,23 @@
+This expense funds the protocol development team for 12 months.
```

Comment:
This should point to the roadmap being funded.
I think it would also be a good place to put in the quarterly update dates.

Reply:
In terms of roadmap - I can include Dave's medium article: https://medium.com/rocket-pool/rocket-pool-houston-upgrade-fa2de86118ad

As I mention in the RPIP, the roadmap will be constantly evolving but I agree the quarterly updates will provide a regular touch-point  - I will include some dates.